### ISSUE TITLE
Podcast page is missing skip link functionality

### ISSUE THREAD

**Issue by aitchiss** [forem/forem#14000]
**Describe the bug**

On the page for an individual podcast - e.g. `https://dev.to/practicaloperations/episode-8-working-from-home` we're missng the `main` landmark with the id "main-content" which allows the skip link functionality to work

**To Reproduce**

1. Go to the Podcasts page and follow one of the podcast links using the keyboard
2. Activate the skip link using the Enter key
3. Try pressing Tab and notice you are not in the main content, but in the header

**Expected behavior**

After pressing the skip link, your next Tab press should show that you are in the main content, and focus the first interactive item (I think this would be the link to the podcast page)

**Screenshots**

https://user-images.githubusercontent.com/20773163/122213182-97b14b00-cea0-11eb-937c-bb68e720ad04.mp4

**Comment by zeotuan** [comment]
Hi can i get assgined to this problem? I think i have found the solution for this.

**Comment by zeotuan** [comment]
@aitchiss i think this issues should be closed

**Comment by aitchiss** [comment]
Closed via #14254

Nice work @zeotuan and thanks for the ping to close off this issue!
